src/ui/dfn-list.js

```diff
@@ -18,6 +18,9 @@ ul.classList.add("respec-dfn-list");
 const render = hyperHTML.bind(ul);
 
 ul.addEventListener("click", ev => {
+  if (ev.target instanceof HTMLElement && !ev.target.matches("a")) {
```

Comment:
This potentially closes the ui when the target is not HTMLElement. It won't happen, but then maybe we don't need that specific condition?

Reply:
Typescript was crying eventTarget has no matches()
Thought this was better than ts-ignore
Open to other ways

Reply:
Oh. How about `if (ev.target instanceof HTMLElement && ev.target.matches("a")) {` and wrapping the code then? Becasue reading this line makes me think whether `Element` is intentionally allowed or not 😂

Reply:
That's cool by me. Updating